openedx_tagging/core/tagging/rest_api/v1/permissions.py

```diff
@@ -0,0 +1,27 @@
+"""
+Taxonomy permissions
+"""
+
+from rest_framework.permissions import DjangoObjectPermissions
+
+
+class TaxonomyObjectPermissions(DjangoObjectPermissions):
+    perms_map = {
+        "GET": ["%(app_label)s.view_%(model_name)s"],
+        "OPTIONS": [],
+        "HEAD": ["%(app_label)s.view_%(model_name)s"],
+        "POST": ["%(app_label)s.add_%(model_name)s"],
+        "PUT": ["%(app_label)s.change_%(model_name)s"],
+        "PATCH": ["%(app_label)s.change_%(model_name)s"],
+        "DELETE": ["%(app_label)s.delete_%(model_name)s"],
+    }
+
+    def has_permission(self, request, view):
+        # Workaround to allow 'retrieve' view to pass through the
+        # method GET permission check. Actual object permission check
+        # is done in the get_object() method.
+        if view.action == "retrieve":
+            return bool(request.user and request.user.is_authenticated)
+        else:
+            return super().has_permission(request, view)
+
```

Comment:
I can see from Jira that you had a question about permissions, but I don't see it on the PR?

Basically, I'm suggesting that we don't need this method at all, and that only `staff` (or `superuser`s) should be allowed to view taxonomies, or do anything else with them.